components/login/provider/__snapshots__/index.test.jsx.snap

```diff
@@ -9,38 +9,38 @@ exports[`ProviderLogin form renders a validation error if login fails 1`] = `
       class="PodBrowser-provider-login__wrapper"
     >
       <div
-        class="PodBrowser-root PodBrowser-selectionBox"
+        class="PodBrowser-root PodBrowser-provider-login__formcontrol"
       >
         <div
           aria-expanded="false"
-          class="PodBrowser-root PodBrowser-hasClearIcon"
+          class="PodBrowser-root PodBrowser-focused PodBrowser-hasClearIcon"
           role="combobox"
         >
           <div
-            aria-describedby="login-error-text"
             class="PodBrowser-root PodBrowser-root PodBrowser-fullWidth"
             data-testid="login-field"
           >
             <div
-              class="PodBrowser-root PodBrowser-root PodBrowser-inputRoot PodBrowser-error PodBrowser-error PodBrowser-fullWidth PodBrowser-formControl PodBrowser-adornedEnd PodBrowser-adornedEnd"
+              class="PodBrowser-root PodBrowser-root PodBrowser-inputRoot PodBrowser-fullWidth PodBrowser-focused PodBrowser-focused PodBrowser-formControl PodBrowser-adornedEnd PodBrowser-adornedEnd"
             >
               <input
                 aria-autocomplete="list"
-                aria-invalid="true"
+                aria-invalid="false"
                 autocapitalize="none"
                 autocomplete="off"
                 class="PodBrowser-input PodBrowser-input PodBrowser-input PodBrowser-inputFocused PodBrowser-inputAdornedEnd PodBrowser-inputAdornedEnd"
                 id="provider-select"
+                inputmode="url"
                 spellcheck="false"
-                type="url"
+                type="text"
```

Comment:
Whilst yes, this has changed to `"text"` note line 33, where it's now using `inputmode="url"` which is what controls the keyboard on mobile being a "url" entry keyboard; This means we don't have the browser execute validation logic on the textbox, as the value may not be a valid "url" (e.g., `inrupt.net`) but that can still be made into a valid url and validated as a IdP.